src/server/handlers.rs

```diff
@@ -761,4 +782,20 @@ impl<P: Publisher> Handler<P> {
         .unwrap_or_default();
         self.send_reply(reply, response).await;
     }
+
+    async fn get_manifest_status(&self, lattice_id: &str, name: &str) -> Option<StatusInfo> {
+        match self
+            .status_stream
+            .get_last_raw_message_by_subject(&format!("wadm.status.{lattice_id}.{name}",))
+            .await
+            .map(|raw| {
+                B64decoder
```

Comment:
Question for my edification: Why do we have to b64 decode the body here?

Reply:
This is because of the `get_last_raw_message` method giving us back b64 encoded values, but if I can set `direct_get` on the stream to `true` I won't have to do this 🤔 Will investigate